files-community/Files
# Fix: Fixed crash that could occur when dragging in grouped grid layout

Since it is difficult to directly fix the crash, scrolling is disabled while dragging as a workaround.

**Resolved / Related Issues**
   Closes #14484 

**Validation**
How did you test these changes?
   1. Change the layout to Grid.
   2. Change the group option to other than None.
   3. Drag an item to the bottom of the window.
   4. No longer crashes.
